app/views/docs/authentication.phtml

```diff
@@ -0,0 +1,278 @@
+}, function (error) {
+    console.log(error); // Failure
+});
+
+</code></pre>
+</div>
+
+<p>
+    An authenticated session in appwrite lasts for 1 year and is then automatically destroyed.
+</p>
+
+<h2><a href="/docs/authentication#persistence" id="persistence">Persistence</a></h2>
+
+<p>
+    Appwrite handles the persistence of the session in a consistent way across SDKs. After authenticating with an SDK, the SDK will persist the session so that the user will not need to log in again the next time they open the app. How the SDK persists in the session depends on the SDK.
+</p>
+
+<ul>
+    <li class="margin-bottom-tiny">Web - Uses a session secure cookie and falls back to local storage when a session cookie is not available.</li>
+    <li class="margin-bottom-tiny">Flutter - Uses a session cookie stored in Application Documents through the path_provider package.</li>
+    <li class="margin-bottom-tiny">Apple - Uses a session cookie.</li>
+    <li class="margin-bottom-tiny">Android - Uses a session cookie stored in SharedPreferences.</li>
+</ul>
+
+<h2><a href="/docs/authentication#security" id="security">Security</a></h2>
+
+<p>
+    Security is very important to protect users' data and privacy. Appwrite uses a permissions model coupled with user sessions to ensure users only have access to certain information based on the permissions. With Appwrite services, including databases and storage, access is granted to documents and files at the collection, bucket, document, or file level. This access is consistent across access to these items in relation to document access, file access, and real-time events.
+</p>
+
+<h2><a href="/docs/authentication#best-practices" id="best-practices">Best Practices</a></h2>
+
+<p>
+    Here are some common best practices when handling sessions and keys used for authentication purposes.
+</p>
+
+<ul>
+    <li class="margin-bottom-tiny">Only keep user sessions active as long as needed.</li>
+    <li class="margin-bottom-tiny">Access Appwrite via HTTPS to prevent any redirects from interfering with your requests.</li>
+    <li class="margin-bottom-tiny">Run Appwrite on standard ports to ensure everything works correctly. The complete endpoint is located in the Appwrite Console</li>
+    <li class="margin-bottom-tiny">Prefer using OAuth2 for authentication with first-party redirects</li>
+    <li class="margin-bottom-tiny">Generate an SSL certification for the Appwrite server</li>
+    <li class="margin-bottom-tiny">Keep API keys private and out of client-side code</li>
+    <li class="margin-bottom-tiny">Only maintain <b>1</b> instance of the Client SDK in your app to avoid conflicting session data</li>
+    <li class="margin-bottom-tiny">When using the realtime service to get users' documents, clean up and resubscribe to the correct resources to maintain the correct app state.</li>
+    <li class="margin-bottom-tiny">Unsubscribing from old resources is also recommended to prevent possible race conditions.</li>
```

Comment:
If we use punctuation for this list item, lets use a period for all list items :)